patch/llpcPatchInOutImportExport.cpp

```diff
@@ -1536,6 +1537,19 @@ void PatchInOutImportExport::visitReturnInst(
                 ++inOutUsage.expCount;
             }
         }
+
+        if (m_pPipelineState->GetBuilderContext()->BuildingRelocatableElf()) {
+          // If we are building relocatable shaders, it is possible there are
+          // generic outputs that are not written to.  We need to count them in
+          // the export count.
+          auto pResUsage = m_pPipelineState->GetShaderResourceUsage(m_shaderStage);
+          for(auto output : pResUsage->inOutUsage.outputLocMap) {
+              if (m_vsExpLoc.count(output.second) != 0) {
+                continue;
+              }
+              ++inOutUsage.expCount;
+          }
+        }
```

Comment:
I hate to be that guy, but the indentation is off here. Here's hoping @trenouf will get the .clangformat going soon.

Reply:
Yes, I probably have a lot of formatting issue.  The .clangformat file will be nice.  I will save everybody a lot of time.